### PR TITLE
Wip/hughsie/pci bcr

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

File: src/fu-util-common.c [hunk 1]
```diff
@@ -1552,7 +1552,9 @@ fu_util_remote_to_string (FwupdRemote *remote, guint idt)
 static void
 fu_security_attr_append_str (FwupdSecurityAttr *attr, GString *str)
 {
-	if (fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+	if (fwupd_security_attr_has_flag (attr, FWUPD_SECURITY_ATTR_FLAG_OBSOLETED)) {
```
Comment (superm1) [inline src/fu-util-common.c:1555]: I'd argue we probably want to sort this list by flags now when printing it.  Success/Fail/Obsoleted.

**Comment by superm1** [comment]
> The kernel approach is preferable, as the config registers can be read even
when the PCI device is hidden from userspace.

This comment is confusing.  Is it really preferable?  It seems to me that it's only preferable in the cases that PCI is missing.  Otherwise this is preferable because it works on older stuff too!

I did check and it works properly on my CML laptop, good find!

File: src/fu-util-common.c (hunk 1 above)
Comment (superm1) [reply src/fu-util-common.c:1555]: I pushed this

**Comment by hughsie** [comment]
> it's only preferable in the cases that PCI is missing

Good point, fixed up.

> but you may check you like my sorting change

Good idea! I moved it to the daemon, I hope that's okay. It's the sort of order I'd want in gnome-firmware too, so it made sense to just return the attrs in a sane order.

**Pull request merged**